Assets/scripts/NoteObject.cs

```diff
@@ -7,6 +7,7 @@ public class NoteObject : MonoBehaviour {
 	private Note _NoteData;
 	public float destroyY = -10f; // Y coordinate under which object will self destruct
 	public Vector3 velocity = new Vector3(0, -0.22f, 0);
+	public Vector3 centerOfObject = new Vector3(.5f, -.5f);

```

Comment:
I changed y to be 100 and it didn't look any different. This is really confusing